src/hrflow_connectors/connectors/poleemploi/connector.py

```diff
@@ -0,0 +1,108 @@
+import typing as t
+
+from hrflow_connectors.connectors.hrflow.warehouse import HrFlowJobWarehouse
+from hrflow_connectors.connectors.poleemploi.warehouse import PoleEmploiJobWarehouse
+from hrflow_connectors.core import BaseActionParameters, Connector, ConnectorAction
+
+
+def get_job_location(poleemploi_location: t.Union[t.Dict, None]) -> t.Dict:
+    if poleemploi_location is None:
+        return dict(lat=None, lng=None, text="")
+
+    lat = poleemploi_location.get("latitude")
+    lat = float(lat) if lat is not None else lat
+
+    lng = poleemploi_location.get("longitude")
+    lng = float(lng) if lng is not None else lng
+
+    concatenate = []
+    for field in ["libelle", "codePostal"]:
+        if poleemploi_location.get(field):
+            concatenate.append(poleemploi_location.get(field))
+
+    return dict(lat=lat, lng=lng, text=" ".join(concatenate))
+
+
+def get_sections(poleemploi_job: t.Dict) -> t.List[t.Dict]:
+    sections = []
+    if (
+        "entreprise" not in poleemploi_job
+        or "description" not in poleemploi_job["entreprise"]
+    ):
+        return sections
+    else:
+        sections.append(
+            dict(
+                name="poleemploi_company_description",
+                title="Company Description",
+                description=poleemploi_job["entreprise"]["description"],
+            )
+        )
+    return sections
```

Comment:
Initialize sections with [] 

Change it only if you need --> add one block of logic 

`    if (
        "entreprise" in poleemploi_job
        and "description" in poleemploi_job["entreprise"]
    ):
                    sections.append(
            dict(
                name="poleemploi_company_description",
                title="Company Description",
                description=poleemploi_job["entreprise"]["description"],
            )
        )
     return sections


`

return sections outside the if block so if you enter it you edit sections if you don't sections is still = [] and you return it either way